csharp/extractor/Semmle.Extraction.CSharp.Standalone/Extractor.cs

```diff
@@ -30,29 +30,46 @@ private static IEnumerable<Action> GetResolvedReferencesStandalone(IEnumerable<s
             IProgressMonitor progressMonitor,
             Stopwatch stopwatch)
         {
-            CSharp.Extractor.Analyse(stopwatch, analyser, options,
-                references => GetResolvedReferencesStandalone(referencePaths, references),
-                (analyser, syntaxTrees) => CSharp.Extractor.ReadSyntaxTrees(sources, analyser, null, null, syntaxTrees),
-                (syntaxTrees, references) => CSharpCompilation.Create(
-                    "csharp.dll", syntaxTrees, references, new CSharpCompilationOptions(OutputKind.ConsoleApplication, allowUnsafe: true)
-                    ),
-                (compilation, options) => analyser.Initialize(compilation, options),
-                () => { },
-                _ => { },
-                () =>
-                {
-                    foreach (var type in analyser.MissingNamespaces)
-                    {
-                        progressMonitor.MissingNamespace(type);
-                    }
+            var output = FileUtils.CreateTemporaryFile(".dll", out var shouldCleanUpContainingFolder);
```

Comment:
Could we risk that anyone relies on us producing a `csharp.dll` for traced extraction?
Maybe we need a change note.

Reply:
`csharp.dll` was only used for standalone extraction, so I think changing it won't cause any major issue for anyone.

Reply:
Oh, of course - you are right.